src/test/java/uk/gov/companieshouse/overseasentitiesapi/mocks/TrustMock.java

```diff
@@ -10,6 +10,7 @@
 public class TrustMock {
     private static void populateCommonTrustDataDao(TrustDataDao trustDataDao) {
         trustDataDao.setTrustName("Trust Name");
+        trustDataDao.setChReference("ch_reference");
```

Comment:
This is really minor but I try to make test data values representative of 'real' possible data. Could you find out what kinds of 'ch_reference' values are stored in the web OE session and use those here (in the SDK PR too)?